src/main/java/com/atlassian/db/replica/internal/SqlFunction.java

```diff
@@ -238,9 +238,6 @@
             "jsonb_to_record",
             "json_to_recordset",
             "jsonb_to_recordset",
-            "currval",
-            "lastval",
-            "nextval",
```

Comment:
Reverting the fix is caught by tests 👍 
`org.postgresql.util.PSQLException: ERROR: cannot execute nextval() in a read-only transaction`